CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 As this project is pre 1.0, breaking changes may happen for minor version bumps. A breaking change will get clearly notified in this log.
 
+## 0.32.0 (Pending)
```

Comment:
Can we add something about gradle? I don't know anything about Java build envs but maybe a note here about the changes would be helpful

Reply:
could mention, but the gradle version upgrade is not an external facing change, there is no developer touchpoint on it, it ends up here as pre-downloaded bins of the new version that developers still run with same cli `gradlew build|test`, they don't need to do additional host system changes for the gradle upgrade.

Reply:
could mention, but the gradle version upgrade is not an external facing change, there is no developer touchpoint on it, it ends up here as pre-downloaded bins of the new version that developers still run with same cli `gradlew build|test`, they don't need to do additional host system changes for the gradle upgrade.